tests/common/test_conf.py

```diff
@@ -17,9 +17,8 @@
 
 import os.path
 
-from azurelinuxagent.common.conf import *
```

Comment:
So stupid, we were importing conf and * at the same time :/
Thanks for cleaning it up! :)